Library/Homebrew/compilers.rb

```diff
@@ -3,6 +3,7 @@
 
 # @private
 module CompilerConstants
+  PREFERRED_GCC = "gcc"
```

Comment:
Think it might be better to use a function for this rather than a constant (as overriding a constant is generally considered a no-no).

Reply:
> I'd suggest overriding gnu_gcc_versions in `extend/os/linux/compilers.rb`. This logic exists only for Linux. This generic version could be changed to def gnu_gcc_versions; GNU_GCC_VERSIONS; end.

I agree with method over constant, but if this logic moved to `extend/os/linux/compilers.rb`, it can remain a constant, as there will be no generic need for `PREFERRED_GCC`.

Reply:
I used a method instead.